tests/spec/core/inlines-spec.js

```diff
@@ -334,6 +335,9 @@ describe("Core - Inlines", () => {
     const iframeAllowAnchor = doc.querySelector("#test2 a");
     expect(iframeAllowAnchor.textContent).toBe("allow");
     expect(iframeAllowAnchor.hash).toBe("#attr-iframe-allow");
+    const inputModeAnchor = doc.querySelector("#test3 a");
+    expect(inputModeAnchor.textContent).toBe(`html-global inputmode "text"`);
```

Comment:
For `[^ link/rel/manifest ^]` will render as `link rel "manifest"`.
Any better textContent?
Should we support renaming like  `[^ link/rel/manifest | link[rel="manifest"] ^]`

Reply:
as discussed, should just be manifest